mutect2-normal-normal.wdl

```diff
@@ -34,26 +34,25 @@ workflow Mutect2NormalNormal {
 	File? pon
 	File? gnomad
 	File? variants_for_contamination
-	Boolean? run_orientation_bias_filter
-	Array[String]? artifact_modes
```

Comment:
@davidbenjamin , is it fine to remove this variable? It wasn't present in the updated mutect2 wdl so the validation workflow didn't run until it removed.

Reply:
Yes you may remove it. I forgot about that one.